x-pack/test/case_api_integration/common/lib/utils.ts

```diff
@@ -589,14 +590,18 @@ export const ensureSavedObjectIsAuthorized = (
 export const createCase = async (
```

Comment:
Looks like we also have a `createCaseAsUser`  do we need the `auth` added here? Could we just use that function instead?

Reply:
This function has been introduced on the RBAC integration tests PR. Now, that we have the `createCase` I plan to delete it on another commit. What I want to avoid, is to have two almost identical function that they differ only on the authorization. Image how many more utility functions we need to create to cover all of our tests.